### PR TITLE
Customize my portfolio main page.

### PR DESCRIPTION
**Pull request by tjgoog**
Demo: https://ttd-step-2020.appspot.com/

Customizations are: Introduce myself, link to my teammates github pages, and showing a photo of my cat.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,26 @@
...
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Tim's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Tim's Portfolio</h1>
```
Comment (megankj) [inline portfolio/src/main/webapp/index.html:11]: this is an arbitrary comment!

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,26 @@
...
+      <p>I am a software engineer at Google. I'm co-hosting three STEP interns this summer with
+          <a href="http://github.com/megankj">Megan</a>. Our pod members are
+          <ul>
+              <li><a href="http://github.com/lilymzhou">Lily</a>
+              <li><a href="http://github.com/ryansimone">Ryan</a>
+              <li><a href="http://github.com/shreyachatterjee00">Shreya</a>
+          </ul>
+      </p>
+      <p>
+          I have a cat and his name is Gunther.
```
Comment (megankj) [inline portfolio/src/main/webapp/index.html:22]: this is a test comment that maybe should be resolved before i approve?

Comment (megankj) [reply portfolio/src/main/webapp/index.html:22]: how do i make this blocking?

Comment (tjgoog) [reply portfolio/src/main/webapp/index.html:22]: I'm not sure if you can.

Comment (megankj) [reply portfolio/src/main/webapp/index.html:22]: how do i un-approve?

**Pull request merged**